tensorboard/plugins/debugger/interactive_debugger_plugin_test.py

```diff
@@ -33,7 +33,7 @@
 import numpy as np
 import portpicker  # pylint: disable=import-error
 from six.moves import urllib  # pylint: disable=wrong-import-order
-import tensorflow as tf  # pylint: disable=wrong-import-order
+import tensorflow  # pylint: disable=wrong-import-order
```

Comment:
Can this be `import tensorflow.compat.v1 as tf` rather than leaving a
lurking (unused) `tensorflow` identifier?

Reply:
I'm worried that this will lead to copybara issues, as I've run into in tensorflow (see CL/264270116). The identifier `tensorflow` is not unused. It's used on line 47.

Reply:
We use `import tensorflow.compat.v1 as tf` extensively in TensorBoard
(you can find existing uses with `git grep`), and it doesn’t cause any
Copybara issues.

> The identifier `tensorflow` is not unused. It's used on line 47.

Of course, but if you change this to `import tensorflow.compat.v1 as tf`
then line 47 goes away, and there are no other uses of `tensorflow`. The
identifier is “lurking” in that `tensorflow` and `tf` are both in-scope
identifiers but have different values, which is confusing.

Reply:
OK. I didn't know `import tensorflow.compat.v1 as tf` was used extensively in TensorBoard. Done.